packages/scene-composer/src/components/three-fiber/ModelRefComponent/GLTFModelComponent.tsx

```diff
@@ -3,6 +3,7 @@ import React, { useContext, useEffect, useMemo } from 'react';
 import { invalidate, ThreeEvent, useFrame, useThree } from '@react-three/fiber';
 import { SkeletonUtils } from 'three-stdlib';
 import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader';
+import { forEach } from 'lodash';
```

Comment:
This probably was added by VSCode trying to help you import it but you don't use it.